po/pl.po

```diff
@@ -399,135 +399,135 @@ msgstr "55xy: Ustaw podtrzymanie obwiedni (x: operator 1-2 (0 dla wszystkich ope
 
 #: src/engine/sysDef.cpp:574
 msgid "10xx: Set global AM depth (0: 1dB, 1: 4.8dB)"
-msgstr ""
+msgstr "10xx: Ustaw głębokość globalną AM (0: 1dB, 1: 4.8dB)"
 
 #: src/engine/sysDef.cpp:578 src/engine/sysDef.cpp:633
 msgid "14xx: Set level of operator 3 (0 highest, 3F lowest)"
-msgstr ""
+msgstr "14xx: Ustaw poziom operatora 3 (0 max., 3F min.)"
 
 #: src/engine/sysDef.cpp:579 src/engine/sysDef.cpp:634
 msgid "15xx: Set level of operator 4 (0 highest, 3F lowest)"
-msgstr ""
+msgstr "15xx: Ustaw poziom operatora 4 (0 max., 3F min.)"
 
 #: src/engine/sysDef.cpp:581
 msgid "17xx: Set global vibrato depth (0: normal, 1: double)"
-msgstr ""
+msgstr "17xx: Ustaw głębokość globalną Vibrato (0: normalna, 1: podwójna)"
 
 #: src/engine/sysDef.cpp:585 src/engine/sysDef.cpp:640
 msgid "1Cxx: Set attack of operator 3 (0 to F)"
-msgstr ""
+msgstr "1Cxx: Ustaw czas narastania w operatorze 3 (0 do F)"
 
 #: src/engine/sysDef.cpp:586 src/engine/sysDef.cpp:641
 msgid "1Dxx: Set attack of operator 4 (0 to F)"
-msgstr ""
+msgstr "1Dxx: Ustaw czas narastania w operatorze 4 (0 do F)"
 
 #: src/engine/sysDef.cpp:587
 msgid ""
 "2Axy: Set waveform (x: operator from 1 to 4 (0 for all ops); y: waveform "
 "from 0 to 3 in OPL2 and 0 to 7 in OPL3)"
-msgstr ""
+msgstr "2Ax7: Ustaw kształt fali (x: operator 1-4 (0 dla wszystkich operatorów); y: kształt fali 0-3 dla OPL2 oraz 0-7 dla OPL3)"
 
 #: src/engine/sysDef.cpp:591 src/engine/sysDef.cpp:659
 msgid "53xy: Set vibrato (x: operator from 1 to 4 (0 for all ops); y: enabled)"
-msgstr ""
+msgstr "53xy: Ustaw vibrato (x: operator 1-4 (0 dla wszystkich operatorów); y: włączone"
 
 #: src/engine/sysDef.cpp:593 src/engine/sysDef.cpp:661
 msgid ""
 "55xy: Set envelope sustain (x: operator from 1 to 4 (0 for all ops); y: "
 "enabled)"
-msgstr ""
+msgstr "55xy: Ustaw podtrzymanie obwiedni (x: operator 1-4 (0 dla wszystkich operatorów); y: włączone)"
 
 #: src/engine/sysDef.cpp:597 src/engine/sysDef.cpp:665
 msgid "59xx: Set decay of operator 3 (0 to F)"
-msgstr ""
+msgstr "59xx: Ustaw zanikanie operatora 3 (0 do F)"
 
 #: src/engine/sysDef.cpp:598 src/engine/sysDef.cpp:666
 msgid "5Axx: Set decay of operator 4 (0 to F)"
-msgstr ""
+msgstr "5Axx: Ustaw zanikanie operatora 4 (0 do F)"
 
 #: src/engine/sysDef.cpp:599 src/engine/sysDef.cpp:667
 msgid ""
 "5Bxy: Set whether key will scale envelope (x: operator from 1 to 4 (0 for "
 "all ops); y: enabled)"
-msgstr ""
+msgstr "5Bxy: Ustaw kiedy klawisz będzie skalował obwiednię (x: operator 1-4 (0 dla wszystkich operatorów); y: włączone)"
 
 #: src/engine/sysDef.cpp:603 src/engine/sysDef.cpp:680
 msgid ""
 "10xx: Set waveform (bit 0: triangle; bit 1: saw; bit 2: pulse; bit 3: noise)"
-msgstr ""
+msgstr "10xx Ustaw kształt fali (bit 0: trójkątna; bit 1: piłokształtna; bit 2: prostokątna; bit 3: szum)"
 
 #: src/engine/sysDef.cpp:604
 msgid "11xx: Set coarse cutoff (not recommended; use 4xxx instead)"
-msgstr ""
+msgstr "11xx: Ustaw zgrubne odcięcie (nie zalecane; zamiast tego użyj 4xxx)"
```

Comment:
"punkt odcięcia" or at least "odcięcie filtra" to be a bit more specific

also, zgrubne? first time hearing this word

Reply:
zgrubny = wstępny, powierzchowny